e2e/test.js

```diff
@@ -50,40 +50,42 @@ test('Game links are different', async t => {
 });
 
 test('Sample Game', async t=> {
-    await t
-        .click(mainPage.createGame)
+    
+    await t.click(mainPage.createGame)
     const opponentLink = await gamePage.joinLink.textContent;
     const myGameURL = await getWindowLocation();
+    
     await t
         .navigateTo(opponentLink)
         .expect(gamePage.cardsInHand.exists).ok()
     const opponentURL  = await getWindowLocation();
+    
     await t
         .navigateTo(myGameURL.href)
         .expect(gamePage.cardsInHand.exists).ok();
+    
     var cardsInPlay = 0;
-    var num = 12;
+    var maxHandNum = await gamePage.cardsInHand.count;
```

Comment:
Great!
That will make changing the number of cards in hand really easy (in case we want that at some point)